src-docs/src/views/tool_tip/tool_tip.js

```diff
@@ -57,7 +57,10 @@ export default () => (
     <EuiSpacer />
 
     <EuiToolTip position="right" content="Works on anything">
-      <EuiFieldText placeholder="Hover over me" />
+      <EuiFieldText
+        placeholder="Hover over me"
+        aria-label="ToolTip appears on hover"
+      />
```

Comment:
Same as above, can we add a visual label to this instead of an `aria-label`?

```suggestion
	<EuiFormRow label="Tooltip demo input">
      <EuiFieldText placeholder="Hover or focus on me" />
    </EuiFormRow>
```